### PR TITLE
feat(app): 640 expand / collapse facet sidebar

### PR DESCRIPTION
**Pull request by alex-wilmer**
Closes #640

### CODE REVIEW

File: app/scripts/search/search.controllers.ts [hunk 1]
```diff
@@ -225,6 +226,11 @@ module ngApp.search.controllers {
       this.$state.go("query." + toTab, stateParams, { inherit: true });
     }
 
+    toggleFacets(shouldCollapse) {
```
Comment (shanewilson) [inline app/scripts/search/search.controllers.ts]: how come this is only needed in search? not projects/annotations ctrl?

Comment (shanewilson) [inline app/scripts/search/search.controllers.ts]: ^ @alex-wilmer

Comment (alex-wilmer) [inline app/scripts/search/search.controllers.ts]: Sorry missed this.. it was only needed to send a message to the pie chart directive to refresh itself, and projects/annotations don't have pie charts.

**Comment by shanewilson** [comment]
👍

**Pull request merged**